RecoHI/HiTracking/python/HILowPtConformalPixelTracks_cfi.py

```diff
@@ -41,6 +41,89 @@
                                         Cleaner = cms.string("trackCleaner")
                                         )
 
+
+###Pixel Tracking -  PhaseI geometry
+
+#Tracking regions - use PV from pp tracking
+from RecoTracker.TkTrackingRegions.globalTrackingRegionWithVertices_cfi import globalTrackingRegionWithVertices
+hiConformalPixelTracksPhase1TrackingRegions = globalTrackingRegionWithVertices.clone(RegionPSet=dict(
+    precise = True,
+    useMultipleScattering = False,
+    useFakeVertices       = False,
+    beamSpot = "offlineBeamSpot",
+    useFixedError = True,
+    nSigmaZ = 3.0,
+    sigmaZVertex = 3.0,
+    fixedError = 0.2,
+    VertexCollection = "offlinePrimaryVertices",
+    ptMin = 0.3,
+    useFoundVertices = True,
+    originRadius = 0.2 
+))
+
+# SEEDING LAYERS
+# Using 4 layers layerlist
+from RecoTracker.IterativeTracking.LowPtQuadStep_cff import lowPtQuadStepSeedLayers
+hiConformalPixelTracksPhase1SeedLayers = lowPtQuadStepSeedLayers.clone()
+hiConformalPixelTracksPhase1SeedLayers.BPix = cms.PSet( 
+         HitProducer = cms.string('siPixelRecHits'),
+         TTRHBuilder = cms.string('WithTrackAngle'),
+)
+hiConformalPixelTracksPhase1SeedLayers.FPix = cms.PSet( 
+         HitProducer = cms.string('siPixelRecHits'),
+         TTRHBuilder = cms.string('WithTrackAngle'),
+)
+
+
+# Hit ntuplets
+from RecoTracker.IterativeTracking.LowPtQuadStep_cff import lowPtQuadStepHitDoublets
+hiConformalPixelTracksPhase1HitDoubletsCA = lowPtQuadStepHitDoublets.clone(
+        seedingLayers = "hiConformalPixelTracksPhase1SeedLayers",
+        trackingRegions = "hiConformalPixelTracksPhase1TrackingRegions"
+)
+
+
+from RecoTracker.IterativeTracking.LowPtQuadStep_cff import lowPtQuadStepHitQuadruplets
+hiConformalPixelTracksPhase1HitQuadrupletsCA = lowPtQuadStepHitQuadruplets.clone(
+	doublets = "hiConformalPixelTracksPhase1HitDoubletsCA",
+        CAPhiCut = cms.double(0.2),
+        CAThetaCut = cms.double(0.0012),
+	SeedComparitorPSet = cms.PSet( 
+           ComponentName = cms.string('none')
+        ),
+        extraHitRPhitolerance = cms.double(0.032),
+        maxChi2 = cms.PSet(
+           enabled = cms.bool(True),
+           pt1 = cms.double(0.7),
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.